app/view/login_logout.py

```diff
@@ -0,0 +1,95 @@
+from flask_jwt_extended import create_access_token
+from flask_jwt_extended import jwt_required
+from app.model.user import User
+from flask import request
+from flask_restful import Resource
+from app.common.ResponseGenerator import ResponseGenerator
+from flask_api import status
+from app.common.logging import *
+from app import db
+from app import jwt
+from datetime import timezone
+from datetime import datetime
+from flask_jwt_extended import get_jwt
+from app.model.login_logout import TokenBlocklist
+import bcrypt
+
+
+class Login(Resource):
+
+    def post(self):
+        try:
+            user_email_id = request.json.get('email_id', None)
+            password = request.json.get('password', None)
```

Comment:
Email and password are mandatory fields, why have you given `None` as default value?
Do not set None here and handle `KeyError` and give respective response.

Reply:
Have removed the None ,because handled the key error